packages/cloud-cognitive/src/components/ProductiveCard/ProductiveCard.stories.js

```diff
@@ -56,14 +56,14 @@ const defaultProps = {
       id: '1',
       icon: Edit16,
       onClick: () => {},
-      onkeydown: () => {},
+      onKeyDown: () => {},
```

Comment:
Might be worth having these use `action('Edit action event')` to ensure these events are working as expected